src/sass/components/_download-intro.scss

```diff
@@ -1,8 +1,16 @@
 .download-intro {
 	@include AU-space( margin-bottom, 2unit );
-	@include AU-space( padding-top, 4unit );
 }
 
 .download-intro--border {
 	@include AU-space( border-top, solid 0.25unit $AU-color-background-shade );
 }
+
```

Comment:
Revert these changes, we use lobotomised owls for spacing of consecutive elements.